winpr/libwinpr/sspi/NTLM/ntlm.h

```diff
@@ -87,6 +88,9 @@ enum _NTLM_AV_ID
 	MsvAvDnsTreeName,
 	MsvAvFlags,
 	MsvAvTimestamp,
+#elif defined(_WIN32)
+    MsvAvTimestamp = 7,
```

Comment:
Formatting

Reply:
I'll take care of this then be back home.